### DISCUSSION TITLE
Error with address that does not exist

### DISCUSSION THREAD

**Discussion by apanyz** [Extended-Thunder/send-later#728] [Q&A] [answered]
Hello, 
I have been using this module for a long time.
A few months ago I sent an email to an address that does not exist by mistake and now every time I send an email I get an error. Always the same error but the email you are trying to send is not in the "send later" folder.
Can you help me delete that email? 
Thank you !!

In the attached images, you can see how I sent 1 email, which is sent correctly and after this, it always tries to send the email to the address that does not exist.

![imagen](https://github.com/user-attachments/assets/252e377c-b046-4808-be46-3bc6c9365fac)

![imagen](https://github.com/user-attachments/assets/98df9f7e-01b3-41e9-ad3f-74e0b9b5c601)

**Comment by jikamens** [comment, marked as answer]
When you schedule a message for delivery with Send Later, at delivery time it copies the message from your Drafts folder into your Outbox and then tells Thunderbird to deliver all messages currently in the Outbox. However, if there's some sort of error in the message that prevents Thunderbird from delivering it, it'll stay stuck in your Outbox and Thunderbird will complain about it every time Send Later delivers any message, since the instruction Send Later gives to Thunderbird is, "Deliver all messages in the Outbox," so it keeps trying to deliver this broken one over and over. So go into Local Folders/Outbox and either delete this broken message or move it back into your Drafts folder so you can fix and resend it.
